arch/arm/core/aarch64/reset.S

```diff
@@ -146,4 +169,20 @@ SECTION_SUBSEC_FUNC(TEXT,_reset_section,__start)
 	/* Enable the SError interrupt */
 	msr	daifclr, #(DAIFSET_ABT)
 
+#if CONFIG_MP_NUM_CPUS > 1
+	mrs     x0, mpidr_el1
+	/* FIMXME: aff3 not taken into consideration */
+	ubfx    x0, x0, #0, #24
+	cbnz    x0, L_enable_secondary
+#endif
+
 	bl	z_arm64_prep_c
+
+#if CONFIG_MP_NUM_CPUS > 1
+L_enable_secondary:
+	adr	x0, arm64_cpu_sp
+	ldr	x1, [x0]
```

Comment:
Note that this ends up with the stack pointers for the secondary cores pointing to the same memory region as the primary core for a bit.

I'd prefer if the initial
```asm
ldr	x0, =(z_interrupt_stacks)
add	x0, x0, #(CONFIG_ISR_STACK_SIZE)
mov	sp, x0
```
grok'd the core ID and set each core's initial stack to its own interrupt stack.

Reply:
@MrVan can you address this?